client/src/pages/SignUpInvitation.vue

```diff
@@ -0,0 +1,172 @@
+          </v-row>
+
+          <v-row>
+            <v-col class="d-flex justify-center">
+              <p class="font-weight-bold text-grey-darken-3">@{{ invitor }}</p>
+              <p class="text-grey-darken-3"> has invited you to join the </p>
+              <p class="font-weight-bold text-grey-darken-3">Test Tracker App</p>
+            </v-col>
+          </v-row>
+
+          <v-row>
+            <v-col class="d-flex justify-center">
+              <p class="text-center text-grey-darken-3">
+                Write your password to confirm your registration
+              </p>
+            </v-col>
+          </v-row>
+
+          <v-row>
+            <v-col>
+              <v-text-field
+                v-model="userPassword.password1"
+                :append-inner-icon="showPassword ? 'mdi-eye-off' : 'mdi-eye'"
+                density="compact"
+                placeholder="Password"
+                prepend-inner-icon="mdi-lock-outline"
+                :rules="passwordRules"
+                :type="showPassword ? 'password' : 'text'"
+                variant="outlined"
+                @click:append-inner="showPassword = !showPassword"
+              />
+              <v-text-field
+                v-model="userPassword.password2"
+                :append-inner-icon="showRePassword ? 'mdi-eye-off' : 'mdi-eye'"
+                density="compact"
+                placeholder="Re Password"
+                prepend-inner-icon="mdi-lock-outline"
+                :rules="confirmedPasswordRules"
+                :type="showRePassword ? 'password' : 'text'"
+                variant="outlined"
+                @click:append-inner="showRePassword = !showRePassword"
+              />
+            </v-col>
+          </v-row>
+
+          <v-row>
+            <v-col class="d-flex justify-center">
+              <v-btn
+                class="mb-8 w-100"
+                color="primary"
+                :disabled="!isFormValid"
+                size="large"
+                @click="RegisterUser"
+              >
+                Register
+              </v-btn>
+            </v-col>
+          </v-row>
+
+          <v-divider />
+          <br>
+
+          <AlreadyHaveAnAccount />
+        </v-form>
+      </v-card>
+    </v-container>
+  </div>
+</template>
+
+<script>
+  import axios from '@/api/axios'
+  import { useRouter } from 'vue-router'
+  import AlreadyHaveAnAccount from '@/components/AlreadyHaveAnAccount.vue'
+  import { useNotifier } from 'vue3-notifier'
+  import { confirmedPasswordRule, passwordRules } from '@/utilities/validators'
+
+  export default {
+    components: {
+      AlreadyHaveAnAccount,
+    },
+    setup () {
+      const router = useRouter()
+      const notifier = useNotifier('top right')
+
+      const userPassword = ref({
+        password1: '',
+        password2: '',
+      })
+
+      const showPassword = ref(true)
+      const showRePassword = ref(true)
+
+      const username = ref('')
```

Comment:
```suggestion
      const firstName = ref('')
```

From where you load the `firstName`?